src/config/locales/en-US.json

```
@@ -1127,7 +1127,12 @@
     "powering_down_hive_power": "Unstaking ({subValue})", 
     "referral_desc": "Invite friends and earn Points", 
     "nft_updateProductName": "NFT Product Name Updated", 
-    "updating": "Updating...", 
+    "updating": "Updating...",
+    "send": "Send",
+    "receive": "Receive",
```

Biome (2.4.11)

[error] The key receive was already declared.

(lint/suspicious/noDuplicateObjectKeys)